design/decisions/why-sql/index.md

```diff
@@ -1,16 +1,194 @@
 ---
-title: "Why SQLite (or other SQL variant)"
-# description: ""
-author: "Kristiane Beicher"
-# date: ""
-draft: true
+author: Kristiane Beicher
 categories:
-  - backend
-  - database
-  - software
-  - tools
-  - technology
+- backend
+- database
+- software
+- tools
+- technology
+draft: true
+title: Why PostgreSQL
 ---
 
-Rationale for why we are choosing the SQL variant, based on guiding
-principles in `design/overview.md`
+# Backend technology
+
+We will at some point be selecting an open source database as the
+backbone of our project. This will support not only the access and
+security features, but it will also be the place where the user will
+ultimately store their data. As the specifications for our system is
+still being developed, we have not yet made the decision of which
+database to work with.
+
+Below we have first detailed the specifications that we are looking for
+in a backend database, and then a discussion of a selection of specific
+systems that may meet our requirements.
+
+# Database specifications
+
+This part of the document is expected to change and grow as we narrow
+down what we will need the seedcase database to do. There are though
+some things that we already know it must be able to handle, and these
+are detailed here.
+
+## Must have functionality
```

Comment:
Or, looking ahead, you mean "we want the database to have these functionality:" and then list them? In that case, you could delete this header and make all subsequent headers level two headers (`##`), since saying specifications in the paragraph above is the same as functionality.